src/utils/handleRequest.test.ts

```diff
@@ -72,6 +72,33 @@ test('returns undefined for a request with the "x-msw-bypass" header', async ()
   expect(callbacks.onMockedResponseSent).not.toHaveBeenCalled()
 })
 
+test('does not bypass a request with "x-msw-bypass" header set to arbitrary value', async () => {
+  const request = createMockedRequest({
+    url: new URL('http://localhost/user'),
+    headers: new Headers({
+      'x-msw-bypass': 'anything',
```

Comment:
Arbitrary values of the `x-msw-bypass` request header do not bypass the request.